wallet/src/de/schildbach/wallet/ui/HeaderBalanceFragment.java

```diff
@@ -18,33 +18,38 @@
 
 import android.app.Activity;
 import android.content.Intent;
-import android.graphics.Color;
+import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.os.Handler;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.View.OnClickListener;
 import android.view.ViewGroup;
+import android.widget.ImageButton;
 import android.widget.ImageView;
 import android.widget.TextView;
 
 import androidx.annotation.NonNull;
 import androidx.core.content.res.ResourcesCompat;
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProvider;
 import androidx.lifecycle.ViewModelProviders;
 import androidx.loader.app.LoaderManager;
 import androidx.loader.content.Loader;
 
 import com.amulyakhare.textdrawable.TextDrawable;
 
 import org.bitcoinj.core.Coin;
+import org.bitcoinj.core.Utils;
```

Comment:
It looks like these imports aren't being used.

Reply:
unused imports removed